### PR TITLE
#382 - cities!

### PR DESCRIPTION
**Pull request by ultimatefiend**

### CODE REVIEW

File: dbmodel.sql [hunk 1]
```diff
@@ -40,10 +40,12 @@ CREATE TABLE IF NOT EXISTS `card` (
...
-  `spot_1` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on top-left, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
-  `spot_2` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-left, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
-  `spot_3` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-middle, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
-  `spot_4` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-right, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock) or NULL for no icon (e.g. special achievement)',
+  `spot_1` TINYINT UNSIGNED DEFAULT NULL COMMENT 'Icon on bottom-left, 0 (hexagon), 1 (crown), 2 (leaf), 3 (bulb), 4 (tower), 5 (factory), 6 (clock), 7 (plus), 8 (flag), 9 (fountain), 10 (echo), 11 (left), 12 (right), 13 (up), 101-111 (1-11 bonus) or NULL for no icon (e.g. special achievement)',
```
Comment (lordmcfuzz) [inline dbmodel.sql:43]: ooo I will have to increase the css for 7 through 13 and create something new for 101-111

File: innovation.game.php [hunk 2]
```diff
@@ -464,12 +464,13 @@ protected function setupNewGame($players, $options = array())
...
         self::initStat('player', 'sharing_effects_number', 0);
         
-        // Remove cards from expansions that are not in use.
-        if (self::getGameStateValue('artifacts_mode') == 1) {
-            self::DbQuery("UPDATE card SET location = 'removed', position = NULL WHERE 110 <= id AND id <= 214");
-        }
-        if (self::getGameStateValue('artifacts_mode') != 3) {
-            self::DbQuery("UPDATE card SET location = 'removed', position = NULL WHERE is_relic");
+        // Add cards from expansions that are in use.
+        if (self::getGameStateValue('artifacts_mode') > 1) {
```
Comment (lordmcfuzz) [inline innovation.game.php:468]: good improvement

File: dbmodel.sql (hunk 1 above)
Comment (ultimatefiend) [reply dbmodel.sql:43]: yeah, we need the sprites too.  Also, the format for the Cities cards is unique to all other Innovation cards so I imagine some CSS would need to change for that too.

Comment (ultimatefiend) [reply dbmodel.sql:43]: #375 has a list of stuff

**Review by micahstairs — APPROVED** [review]
Thanks! We can confirm that each of the cards have the right icons when we've actually rendered them onscreen.

**Pull request merged**